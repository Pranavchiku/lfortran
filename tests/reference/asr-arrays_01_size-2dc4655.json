{
    "basename": "asr-arrays_01_size-2dc4655",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/arrays_01_size.f90",
    "infile_hash": "cb58e1bbc81b8cb3e39e493299f8ce73339c3bfe6f97ec12ea102145",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-arrays_01_size-2dc4655.stdout",
    "stdout_hash": "1b9e15bd8a52711ced65a7d01614ae7feff68b84dea8bfc89d39a1ea",
    "stderr": "asr-arrays_01_size-2dc4655.stderr",
    "stderr_hash": "19a261cdb9374fe5bad2e1b332b31b1d8e56ccc27d2bd2647207a526",
    "returncode": 0
}