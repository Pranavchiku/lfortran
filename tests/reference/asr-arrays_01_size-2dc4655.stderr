warning: Implied save feature
 --> tests/../integration_tests/arrays_01_size.f90:3:1
  |
3 | integer :: i, a(3), b(4)
  | ^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/arrays_01_size.f90:3:1
  |
3 | integer :: i, a(3), b(4)
  | ^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/arrays_01_size.f90:3:1
  |
3 | integer :: i, a(3), b(4)
  | ^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/arrays_01_size.f90:4:1
  |
4 | integer :: size_a
  | ^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/arrays_01_size.f90:5:1
  |
5 | integer :: size_b
  | ^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
